ldap_jwt_auth/routers/refresh.py

```diff
@@ -0,0 +1,37 @@
+"""
+Module for providing an API router which defines a route for managing the refreshing/updating of a JWT access token
+using a JWT refresh token.
+"""
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Cookie, Depends, HTTPException, Query, status
+from fastapi.responses import JSONResponse
+
+from ldap_jwt_auth.auth.jwt_hanlder import JWTHandler
+from ldap_jwt_auth.core.exceptions import JWTRefreshError
+
+logger = logging.getLogger()
+
+router = APIRouter(prefix="/refresh", tags=["authentication"])
+
+
+@router.post(
+    path="/",
+    summary="Generate an updated JWT access token using the JWT refresh token",
+    response_description="A JWT access token",
+)
+def refresh_access_token(
+    jwt_handler: Annotated[JWTHandler, Depends(JWTHandler)],
+    access_token: Annotated[str, Query(description="The JWT access token to refresh")],
```

Comment:
This needs to be named `token` in the body for SciGateway (See https://github.com/ral-facilities/operationsgateway-api/blob/fb655d976e794e23a96828f6d2ad8cdef46d0141/operationsgateway_api/src/routes/auth.py#L18-L24)

Reply:
Addressed in https://github.com/ral-facilities/ldap-jwt-auth/pull/11/commits/886f9c1f5115a369688a0d31c3c151566bc6f93a